test/unit/mesh/mesh.js

```diff
@@ -721,7 +723,8 @@ describe(tests.testName(__filename, 3), function() {
       testMethod1() {}
       testMethod2() {}
       __testMethod3() {}
-      testMethod4() {}
+      // eslint-disable-next-line no-unused-vars
+      testMethod4($happn) {}
```

Comment:
Can you add the async keyword?

Reply:
we do have tests for that elsewhere, but will add a case here too